src/datanode/src/datanode.rs

```diff
@@ -458,20 +458,33 @@ impl DatanodeBuilder {
     async fn build_mito_engine(
         opts: &DatanodeOptions,
         object_store_manager: ObjectStoreManagerRef,
-        config: MitoConfig,
+        mut config: MitoConfig,
     ) -> Result<MitoEngine> {
+        // Sets write cache path if it is empty.
+        if config.experimental_write_cache_path.is_empty() {
+            config.experimental_write_cache_path = join_dir(&opts.storage.data_home, "write_cache");
```

Comment:
> Write cache is disabled if the path is empty.

But you modify it...

Reply:
I struggled with it. I want to put the write cache under our data home by default. We don't know what data_home is so I can't give it a default value in `MitoConfig::default()`. However, others can all `MitoEngine::new()` with an empty write path. At that time I couldn't decide the write cache path and must disable the cache.

Reply:
Added an enable flag to control the write cache.